lib/note-detail.jsx

```diff
@@ -60,6 +68,21 @@ export default React.createClass( {
 		}
 	},
 
+	onEditorChanged: function() {
+		clearTimeout( saveTimerId );
+		// Save the note after user stops typing
+		saveTimerId = delay( this.saveNote, saveDelay );
+	},
```

Comment:
Not sure exactly what behavior you are seeing, @roundhill, but I made a quick [example on jsbin](http://jsbin.com/jeyoxibula/3/edit?js,console)